### PR TITLE
Fix typos in the styish formatter documentation

### PR DESCRIPTION
**Pull request by dguo**
#### PR checklist

- [ ] Addresses an existing issue: #0000
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [x] Documentation update

#### Overview of change: 

Fixes typos.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/tslint](https://github.com/palantir/tslint), @dguo! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: docs/formatters/stylish/index.html [hunk 1]
```diff
@@ -3,13 +3,13 @@
 description: Human-readable formatter which creates stylish messages.
 descriptionDetails: |-
 
-  The output matches that produced by eslint's stylish formatter. Its readability
-  enhanced through spacing and colouring
+  The output matches what is produced by eslint's stylish formatter. Its
+  readability is enhanced through spacing and colouring.
```
Comment (ajafff) [inline docs/formatters/stylish/index.html]: this file is not meant to be manually edited. the content are generated from the metadata in the source file:
https://github.com/palantir/tslint/blob/master/src/formatters/stylishFormatter.ts#L32

Comment (dguo) [reply docs/formatters/stylish/index.html]: Thanks. I pushed a new commit.

**Comment by ajafff** [comment]
Thanks @dguo

**Pull request merged**